### PR TITLE
Added code for angaram

### PR DESCRIPTION
**Pull request by Sathwika-02**
@tarun26091999 @rudrakshi99  please review this. This will solve issue #395

### CODE REVIEW

**Comment by krishna-NIT** [comment]
@Sathwika-02  this 395 issue is not linked properly, add space in betweeen # there 
means #395

**Comment by krishna-NIT** [comment]
@Sathwika-02  this 395 issue is not linked properly, add space in betweeen # there
means space
image

**Comment by rish-singhal** [comment]
please resolve the conflicts.

File: Code/Python/Anagram.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    An anagram of a string is an  another string that contains the same characters but the order of characters can be different.
+For example ,listen and silent are anagrams of each oter because all the characters in the string "listen" are present in string "silent".
+So, listen and silent are anagrams of each other.
+ 
+"""
+
+# Solution:
+def anagram(string1,string2):
+
+#convert string1 to list of characters
```
Comment (rish-singhal) [inline Code/Python/Anagram.py:14]: indent these comments with tabs

File: Code/Python/Anagram.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+For example ,listen and silent are anagrams of each oter because all the characters in the string "listen" are present in string "silent".
+So, listen and silent are anagrams of each other.
+ 
+"""
+
+# Solution:
+def anagram(string1,string2):
+
+#convert string1 to list of characters 
+     list_string1=list(string1)
```
Comment (rish-singhal) [inline Code/Python/Anagram.py:15]: give spaces around =, +, ... operators

File: Code/Python/Anagram.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+#convert string2 to a list of characters
+
+     list_string2=list(string2)
+
+#sort list of characters
+
+     list_string2.sort()
+
+     if(list_string1==list_string2):
```
Comment (rish-singhal) [inline Code/Python/Anagram.py:28]: here also, spaces around ==

File: Code/Python/Anagram.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+#Take two strings as input from the user 
+
+string1=input("enter string1:")
+string2=input("enter string2:")
+anagram(string1,string2)
+
+
+"""
+
+Time Complexity: O(nlogn)
```
Comment (rish-singhal) [inline Code/Python/Anagram.py:42]: what is n? please describe it here

File: Code/Python/Anagram.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  
+  Output: Given strings are anagrams of eachother
+
+ Input:
+ enter string1: hello
+ enter string2: world
+ Output: Given strings are not anagrams of eachother.
+
+"""
+
```
Comment (rish-singhal) [inline Code/Python/Anagram.py:58]: remove these extra lines

**Review by rish-singhal — CHANGES_REQUESTED** [review]

**Pull request closed without merging**